easybuild/easyconfigs/l/libpng/libpng-1.6.30-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.30'
```

Comment:
@JackPerdue We already have an easyconfig for `libpng` 1.6.34, there's no point in adding one for an older version?